azure-cognitiveservices-vision-computervision/sdk_packaging.toml

```diff
@@ -4,3 +4,4 @@ package_pprint_name = "Cognitive Services Computer Vision"
 package_doc_id = "cognitive-services"
 is_stable = false
 is_arm = false
+autoupdate = false
```

Comment:
It's actually `auto_update` (my mistake, sorry)

Reply:
@diberry Looks like we have a merge conflict and need [another edit](https://github.com/Azure/azure-sdk-for-python/pull/4255#discussion_r261723085) (per @lmazuel). I can issue a PR into yours, or take all these changes and open a new PR under my account, then shepherd it through the process without needing to bug you again. :smile: Happy do do either--what's your preference?